src/main/java/it/gov/pagopa/xpay/controller/XPayController.java

```diff
@@ -122,4 +137,31 @@ private String hashMac(String realMac) throws Exception {
         }
         return builder.toString();
     }
+
+    private XPayErrorEnum getErrorConfig() {
+        String errorCode = configRepository.findByPropertyKey(XPAY_AUTH_ERROR).getPropertyValue();
+
+        try {
```

Comment:
@umberto-russo-nexi Duplicated logic: you can use EnumUtils.isValidEnum() --> see comment on src/main/java/it/gov/pagopa/xpay/controller/XPaySettingsController.java